src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -891,7 +891,7 @@ There are various error codes for different error scenarios. The following table
 | E1070 | Could not find OrganisationUnit: `{0}`, linked to Enrollment. | The system could not find the OrganisationUnit specified in the Enrollment payload. |
 | E1074 | FeatureType is missing. | |
 | E1075 | Attribute: `{0}`, is missing uid. | |
-| E1076 | Attribute: `{0}`, value is null. | |
+| E1076 | `{0}` `{1}` is mandatory and can''t be null | |
```

Comment:
I see the code also has the same typo as here (can''t); Instead of merging this, then fixing it in the code and updating the docs again. I think it's better to write it correctly here, and update the code after :)